database/src/main/resources/db/migration/V1_174__service_search_result_external_interface_id.sql

```diff
@@ -0,0 +1,49 @@
+ALTER TYPE external_interface_search_result
+    ADD ATTRIBUTE id INTEGER;
+
+DROP VIEW transport_service_search_result;
+
+-- Re-create view to include external-interface-description id to external_interface_search_result
+CREATE VIEW transport_service_search_result AS
+SELECT t.*,
+       op.name as "operator-name",
+       op."business-id" as "business-id",
+       -- Changed Start
```

Comment:
move these comments to row 23 and end to 31

Reply:
Ok